SDL2pp/Surface.cc

```diff
@@ -41,16 +41,33 @@ Surface::Surface(SDL_Surface* surface) : surface_(surface) {
 	assert(surface);
 }
 
-Surface::Surface(Uint32 flags, int width, int height, int depth, Uint32 Rmask, Uint32 Gmask, Uint32 Bmask, Uint32 Amask) {
-	if ((surface_ = SDL_CreateRGBSurface(flags, width, height, depth, Rmask, Gmask, Bmask, Amask)) == nullptr)
+Surface::Surface(int width, int height, int depth, Uint32 Rmask, Uint32 Gmask, Uint32 Bmask, Uint32 Amask) {
```

Comment:
SDL does doesn't say it's obsolete. Flags could be unused now, but will be at some point. Also, breaking API is bad, m'kay. Could add a thin wrapper, but I don't think the complication is worth this single argument.